standard/expressions.md

```diff
@@ -6201,6 +6368,49 @@ When a property or indexer declared in a *struct_type* is the target of an assig
 >
 > *end example*
 
+### §ref-assignment-new-clause Ref assignment
+
+The `= ref` operator is known as the *ref assignment* operator.
+
+The left operand shall be an expression that binds to a reference variable (§ref-span-safety), a reference parameter (other than `this`), an output parameter, or and input parameter. The right operand shall be an expression that yields a *variable_reference* designating a value of the same type as the left operand.
+
+It is a compile time error if the ref-safe-context of the left operand is wider than the ref-safe-context of the left operand (§ref-safe-contexts).
```

Comment:
Should one of these lefts be right?  And the other wrong?

Reply:
There should be a cross-reference to the definition of ref-safe-context.

Reply:
Well the second left should be right, that's a typo, but I'm lost on the other one being wrong! 😉

There is a cross-reference, are you suggesting it be moved to be after “ref-safe-context” itself and not placed at the end of the sentence? Makes sense to me.

```suggestion
It is a compile time error if the ref-safe-context (`§ref-safe-contexts`) of the left operand is wider than the ref-safe-context of the right operand.
```